hello-world.cc

```diff
@@ -0,0 +1,8 @@
+#include <ctime>
```

Comment:
Add top-level comment documenting that this is an example file that will be replaced with the actual fuzz target rule functionality.